docs/user-guide.md

```diff
@@ -419,18 +419,24 @@ spec:
 Note that the mutating admission webhook is needed to use this feature. Please refer to the
 [Quick Start Guide](quick-start-guide.md) on how to enable the mutating admission webhook.
 
-### Using Pod Security Context
+### Using Security Context
 
-A `SparkApplication` can specify a `PodSecurityContext` for the driver or executor pod, using the optional field `.spec.driver.securityContext` or `.spec.executor.securityContext`. Below is an example:
+A `SparkApplication` can specify a `PodSecurityContext` or `SecurityContext` for the driver or executor pod/container, using the optional field `.spec.driver.PodsecurityContext` or `.spec.driver.securityContext` or `.spec.executor.PodsecurityContext` or `.spec.executor.securityContext`. Below is an example:
```

Comment:
You might want to split this into two parts: one for `PodSecurityContext` and one for `SecurityContext`.

Reply:
Split the doc, and kept the YAML file together since there are cases where people want to set the context and overwrite in the container level.